### ISSUE TITLE
Getting error '-1'

### ISSUE THREAD

**Issue by zeamrl** [NeielHQ/cordova-plugin-ussd#2]
Hi,
I'm getting the error -1 when I tried to request a common code from my mobile phone operator.
What does that mean?

**Comment by LimitlessDonald** [comment]
Can you kindly let me know the price code you tried ?.
Thanks

**Comment by zeamrl** [comment]
Yes, I tried with USSD codes *111# and *555# and both don't work and return the same error.
Thanks!

**Comment by zeamrl** [comment]
Do you have any updates on it?

**Comment by LimitlessDonald** [comment]
> 

Hi, sorry for the late response, I have been tied up with a lot of projects. did you get any progress with this ?
